README.md

```diff
@@ -184,6 +184,15 @@ Outputs.
 <outsider/>
 ```
 
+If otherwise you need `standalone ="false"` in the XML declaration, you can pass `standalone: false` as an option to the` generate / 2` call.
+
+Outputs.
+
+```xml
+<?xml version="1.0" standalone="false"?>
```

Comment:
Did you mean `standalone="no"`? I believe valid values are "yes" or "no"

Reply:
My bad 😅